addon/mixins/sortable-item.js

```diff
@@ -377,17 +384,16 @@ export default Mixin.create({
     @return Promise
   */
   _waitForTransition() {
-    return new Promise(resolve => {
-      run.next(() => {
-        let duration = 0;
-
-        if (this.get('isAnimated')) {
-          duration = this.get('transitionDuration');
-        }
-
-        run.later(this, resolve, duration);
+    if (this.get('isAnimated')) {
```

Comment:
This lets the non-async test run smoothly by having the animations disabled. Hopefully this doesn't introduce anything bad.